src/components/Header.js

```diff
@@ -1,28 +1,41 @@
-import React, { Component } from 'react';
-import '../App.css';
+import React, { Component } from "react";
+import "../App.css";
 import Navbar from "react-bootstrap/Navbar";
 import Nav from "react-bootstrap/Nav";
 import NavDropdown from "react-bootstrap/NavDropdown";
 class Header extends Component {
```

Comment:
You are only rendering elements in this component. You don't need to make it a class component.

Reply:
ok fine